sync/sync-impl/src/test/java/com/duckduckgo/sync/impl/AppSyncRepositoryTest.kt

```diff
@@ -192,6 +197,20 @@ class AppSyncRepositoryTest {
         verify(syncStore, times(0)).clearAll()
     }
 
+    @Test
+    fun whenLogoutRemoteDeviceSucceedsThenReturnSuccessButDoNotRemoveLocalData() {
```

Comment:
since you have `deviceId` as a parameter that can be empty, there's a test missing when `logout` is called without `deviceId`.

Reply:
I've removed the default value, and now `deviceId` is required 👍